src/actions/nodePoolsActions.js

```diff
@@ -8,10 +8,11 @@ const nodePoolsApi = new GiantSwarm.NodepoolsApi();
 //Loads all node pools for all node pools clusters.
 export function nodePoolsLoad() {
   return async function(dispatch, getState) {
-    const clusters = getState().entities.clusters.nodePoolsClusters || [];
+    const nodePoolsClustersId =
```

Comment:
Do we really do `nodePoolsClustersId`, not `nodePoolsClustersID` ?

Reply:
What do you mean? Do we have any format or styling rule for this? I think I always have written `id`...

Reply:
Not really a rule yet. Worth discussion I think. govet reminds devs to uppercase abbreviations like "ID", "API" in variable names, so e. g. `myAPIEndpoint` instead of `myApiEndpoint`. If you want `nodePoolsClustersId`, ignore me.